bgpd/bgpd.h

```diff
@@ -1053,6 +1053,41 @@ struct llgr_info {
 	uint8_t flags;
 };
 
+struct peer_connection {
```

Comment:
I'm thinking maybe ttl/rtt/etc. are kinda related to "connection" or do we care here only about the thread/IO?

Reply:
Mostly care about for the thread/IO.  But am willing to move stuff around .  Long term goal is to remove the doppelganger.  This is the start of such work.